configs/defs/generic_linux.textproto

```diff
@@ -9558,3 +9558,931 @@ benchmark_configs: {
+      "check_alternatives:{"
+      "  file_checks:{"
+      "    files_to_check:{single_file:{path:\"/bin\"}}"
+      "    existence:{should_exist: false}"
+      "  }"
+      "}"
+      "}"
+  }
+}
+
+
+
+benchmark_configs: {
+  id: "ensure-audit-configuration-files-group-configured"
+  compliance_note: {
+    title: "Ensure audit configuration files owner is configured"
+    description:
+      "Access to the audit configuration files could allow unauthorized personnel to prevent the "
+      "auditing of critical events.\n"
+      "Misconfigured audit configuration files may prevent the auditing of critical events or "
+      "impact the system's performance by overwhelming the audit log. Misconfiguration of the "
+      "audit configuration files may also make it more difficult to establish and investigate "
+      "events relating to an incident."
+    remediation:
+      "Run the following command to set permissions on audit configuration files:\n"
+      "```\n"
+      "find /etc/audit/ -type f \\\\( -name '*.conf' -o -name '*.rules' \\\\) -exec chmod u-x,g-wx,o-rwx {} +\n"
+      "```"
+  cis_benchmark: {
+    profile_level: 2
+    severity: MEDIUM
+  }
+  scan_instructions:
+    "generic:{check_alternatives:{"
+    "    file_checks:{"
+    "      files_to_check:{single_file:{path:\"/etc/audit/auditd.conf\"}}"
+    "      existence:{should_exist:true}"
+    "    }"
+    "    file_checks:{"
+    "      files_to_check:{single_file:{"
+    "        path:\"/etc/audit/auditd.conf\""
+    "      }}"
+    "      permission:{"
+    "        group: {name: \"root\" should_own: true}"
+    "      }"
+    "    }"
+    "  }"
+    "}"
+  }
+}
+
+
+
+benchmark_configs: {
+  id: "ensure-audit-configuration-files-mode-configured"
+  compliance_note: {
+    title: "Ensure audit configuration files mode is configured"
+    description: "Audit configuration files must be configured with mode 0640 or more restrictive to prevent unauthorized "
+    "access and protect the integrity of the audit system."
+    rationale: "Restricting access to audit configuration files ensures that only authorized personnel can modify the "
+    "settings, preventing the disabling of critical auditing or misconfigurations that may impact performance or "
+    "investigation efforts."
+    remediation:
+      "Run the following command to set permissions on audit configuration files:\n"
+      "```bash\n"
+      "find /etc/audit/ -type f \\\\( -name '*.conf' -o -name '*.rules' \\\\) -exec chmod u-x,g-wx,o-rwx {} +\n"
+      "```\n"
+  cis_benchmark: {
+    profile_level: 2
+    severity: MEDIUM
+  }
+  scan_instructions:
+    "generic:{check_alternatives:{"
+    "      file_checks:{"
+    "        files_to_check:{files_in_dir:{"
+    "          dir_path:\"/etc/audit/\""
+    "          recursive: false"
+    "          files_only: true"
+    "          filename_regex: \".*\\\\.(conf|rules)\""
+    "        }}"
+    "        permission:{"
+    "          set_bits: 0640"
```

Comment:
We can ignore the clear bits since it the benchmark says "0640 or more restrictive". This currently only passes if the permissions are exactly 0640.